package/base-files/Makefile

```diff
@@ -110,18 +110,18 @@ Build/Compile = $(Build/Compile/Default)
 
 ifdef CONFIG_SIGNED_PACKAGES
   define Build/Configure
-	[ -s $(BUILD_KEY) -a -s $(BUILD_KEY).pub ] || \
-		$(STAGING_DIR_HOST)/bin/usign -G -s $(BUILD_KEY) -p $(BUILD_KEY).pub -c "Local build key"
+	[ -s $(BUILD_KEY).sec -a -s $(BUILD_KEY).pub ] || \
```

Comment:
> FYI signify: can't open /opt/devel/openwrt/openwrt.git/key-build.pub for writing: File exists

The problem happens here. In existing setups the files `key-build` and `key-build.pub` should exists, signify however requires `key-build.sec` and tries to create both. A migration step would be required to *move* `key-build` to `key-build`. 

```bash
        #migrating key-build over to key-build.sec
        [ -s $(BUILD_KEY).sec -a ! -s $(BUILD_KEY) ] || \
                mv $(BUILD_KEY) $(BUILD_KEY).sec
```

This could expose peoples private key when they have scripts securing `key-build` but not `key-build.sec`, e.g. store it accidently as an artifact.

Creating a symlink does not work, at least not the way I tried. Signify would complain:

```bash
[a@tb openwrt (signify)]$ signify-openbsd -S -s key-build.sec -x test.sig -m Makefile
signify-openbsd: can't open key-build.sec for reading: Too many levels of symbolic links
[a@tb openwrt (signify)]$ ls -l key-build*
-rw------- 1 a a 187 Apr 20 09:06 key-build
-rw-r--r-- 1 a a 103 Apr 20 09:06 key-build.pub
lrwxrwxrwx 1 a a   9 Apr 20 09:42 key-build.sec -> key-build
```

The script I tried:

```bash
        #migrating key-build over to key-build.sec
        [ -s $(BUILD_KEY).sec -a ! -s $(BUILD_KEY) ] || \
                ln -s $(BUILD_KEY) $(BUILD_KEY).sec
```

Reply:
@jow- suggested to simply throw an error message and let the user do the fixing manually. I think that does make sense in case some scripting is used.

Reply:
> A migration step would be required to move `key-build` to `key-build.`

Err... move it to `key-build.sec` you mean?

> This could expose peoples private key when they have scripts securing `key-build` but not `key-build.sec`, e.g. store it accidently as an artifact.

Could you check in `.gitignore` for a whitelist rule?

Reply:
In .gitignore is `key-build*` which also ignores `key-build.sec`. I therefore suggest to move instead.